src/server.js

```diff
@@ -205,11 +206,15 @@ const main = async () => {
      * used by all other Bard functions. See SUP-686 for more detail.
      */
     const response = await res.json()
-    const email = _.get('value', _.find({ key: 'anonymousGroup' }, response.keyValuePairs))
+    const anonEmail = _.get('value', _.find({ key: 'anonymousGroup' }, response.keyValuePairs))
+    const realEmail = _.get('value', _.find({ key: 'email' }, response.keyValuePairs))
     const data = {
       '$token': token,
       '$distinct_id': userDistinctId(req.user),
-      '$set': { '$email': email }
+      '$set': {
+        '$email': anonEmail,
+        '$accountType': getAccountType(realEmail)
```

Comment:
I think the `$` prefix is for specific properties that Mixpanel recognizes. Therefore, I think this should be `accountType`. However, I'm having trouble finding confirmation of this in the Mixpanel docs.

Reply:
I suggest doing some experiments and looking at what comes through in Mixpanel an decide what to do from there.

Reply:
Or... maybe you already did this because I just noticed that there's a separate commit where you added the `$`. 🤷‍♂️

Reply:
I did some experiments on dev